src/overlays/actors/ovl_En_Insect/z_en_insect.c

```diff
@@ -148,105 +162,108 @@ s32 EnInsect_FoundNearbySoil(EnInsect* this, GlobalContext* globalCtx) {
     return ret;
 }
 
-void func_80A7C058(EnInsect* this) {
-    if (this->unk_31E > 0) {
-        this->unk_31E--;
+/**
+ * Update the crawl sound timer, and play the crawling sound when it reaches 0.
+ */
+void EnInsect_UpdateCrawlSfx(EnInsect* this) {
+    if (this->crawlSoundTimer > 0) {
+        this->crawlSoundTimer--;
         return;
     }
 
     Audio_PlayActorSound2(&this->actor, NA_SE_EN_MUSI_WALK);
```

Comment:
we can maybe name it `crawlSoundDelay`? the current name makes it sound like the sound plays while the timer is non zero